examples/WriteExample.h

```diff
@@ -0,0 +1,33 @@
+/*
```

Comment:
The previous version with the single `Examples.h` header looked better. Would it make sense to merge this header back to the `Examples.h`?